build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+make clean
+cd rust
+rm Cargo.lock
+cargo update -p "filecoin-proofs-api"
+cargo install cbindgen
```

Comment:
@cryptonemo - Concern: I don't think that the `cargo install cbindgen` command [will respect](https://github.com/rust-lang/cargo/issues/7169) the version of cbindgen specified in our our Cargo.lock file, which means that `cargo build` (which uses `build.rs`) could produce different output than the output produced by the `cbindgen` binary. In rare cases, this may result in CircleCI builds performing differently than local builds.

Reply:
Ah, interesting.  I added it to solve the reverse problem (Cargo.lock specified a version too old to succeed for me, so I updated Cargo.toml).  If I just remove `cargo install cbindgen`, it should still work, no?

Reply:
Actually, this script isn't for CI, it's for the end user, so I don't think it should matter (other than perhaps not be needed).  I needed it for some reason though.